set.mm

```diff
@@ -510554,6 +510819,1209 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
+      TZUWLWRWTVXQVXSVYFUXMUVJWVNWEVXQWXLWXPXUEXUJUVIWYPVXQBCDEFGUVJUXJVXIUXKVX
+      JHIJKLMWUIWUMWULWVDWVAWURWXQYNZTWTVXQUXCUXMVWTVXSUVJWVOWVSTVXQWXJWXLXUAXU
+      EUVIWYRVXQUXCUXMVWOVXKUVJWVOWVRTZTWTWWAXCWWBXCWWCXCVWRUXNVUOVWOUVJSZXUAWX
+      TUVISZVTZUEUFBFVXEVVDXVPUGUHBFVXHVXLXVPUIUNBFVXQVVJVXKUVJSZXUEWYDUVISZXVN
+      XVOVXQVVRVXKUVJSZXUJWYHUVISZXVQXVRVXQVVOVXIGSZVYKPZDQXUHVYEGSZWYGXUIGSZCS
+      ZXUIVYEGSZPZDQXVSXVTVXQXWBXWGDUVGXUSVXQBDEFVXJGXWBVVMVXIGSZVVNVXIGSZCSZVX
+      JXWJGSZVYKVYKVXJGSZXWGIMWUIWUMJVXQXWAXWJVYKVXQBCEGVVMVVNVXIIKJWUKWUQWVCWV
+      DYKYLVXQXWEXWKXWFXWLVXQXWEVXJXWHGSZVXJXWIGSZCSXWKVXQXWCXWMXWDXWNCVXQXWHVX
+      JGSVVMVXIVXJGSZGSXWMXWCVXQBEGVVMVXIVXJIJWUKWUQWVDWUSYIVXQBEGVXJXWHIJWUIWU
+      SVXQBEGVVMVXIIJWUKWUQWVDXRZYJVXQBGUVSVXJUXJVXIVVBUWCVWJXVGWULWVDWUPWUSYOY
+      SVXQXWIVXJGSVVNXWOGSXWNXWDVXQBEGVVNVXIVXJIJWUKWVCWVDWUSYIVXQBEGVXJXWIIJWU
+      IWUSVXQBEGVVNVXIIJWUKWVCWVDXRZYJVXQBGUVSVXJVVAVXIUXKUWCVWJXVGWUTWVDWVBWUS
+      YOYSTVXQBCEGVXJXWHXWIIKJWUKWUSXWPXWQYQYTVXQVVBXUIGSZVXJGSVVBXUIVXJGSGSXWL
+      XWFVXQBEGVVBXUIVXJIJWUKWUPVXQFBXUIWUNXVKWHZWUSYIVXQVYKXWRVXJGVXQVYKVVBUXK
+      GSZVXJGSXWRVXQVVPXWTVXJGVXQBEGUXKVVBIJWUIWVBWUPYJXEVXQBEGVVBUXKVXJIJWUKWU
+      PWVBWUSYIWOXEVXQBEGXUIVVBVXJIJWUIXWSWUPWUSYPYSWQVXQBCEXWHXWIIKWUJXWPXWQXS
+      ZVXQBEGVXJXWJIJWUKWUSXXAXRVXQVYKVYPFWVHXVIYAZVXQGFUVSVYKVXJVWJWUMXXBWURXT
+      WURVXQXWKVGVXQBEGVYKVXJIJWUIVXQFBVYKWUNXXBWHWUSYJYRYMVXQBCDEFGUVJVVOVXIVV
+      PVXJHIJKLMWUIWUMWVIWVDWVJWURWXQYNVXQBCUVIDEFGXUHWYGXUIVYEHIJKLMWUIWUMXVJW
+      YOXVKWVLUWLWRWTVXQVVJVVRVXKUVJWVMXEVXQXUEXUJWYDWYHUVIXVLWYQTWTVXQVUOVVJVW
+      OVXKUVJWVQWVRTVXQXUAXUEWXTWYDUVIXVMWYSTWTWWAXCWWBXCWWCXCAUVLUVGRUVMUVHRAU
+      VKUVKBFAFBUVKUWEUWTWHUWTVNUVGUVLDUXAVPVHUXIUXNUVMUXCUVJSZUXCVTUEUFBFUXSXX
+      CUVMUXMUVJSZUXCUXSUXCUXMUVMUVJUYAWEUXSUVKUXJGSZUYFPZDQUXMXXDUXCUXSXXFUXLD
+      UXSXXEUXJUYFUXKUXSBEGUVKUXJIJUWRUYHUYNWPUYRWQWSUXSBCDEFGUVJUVKUXJUVKUXKHI
+      JKLMUYTUYIUXSFBUVKUYJVUBWHZUYNVUBUYKWXQYNUYAWTWOVUCXCUXIUXNUXCUVMUVJSZUXC
+      VTUEUFBFUXSXXHUXMUXCUXSUXMUVMUVJSUYCVUHPZDQXXHUXMUXSBCDEFGUVJUXJUVKUXKUVK
+      HIJKLMUYTUYIUYNXXGUYKVUBWXQYNUXSUXCUXMUVMUVJUYAXEUXSUXLXXIDUXSUXJUYCUXKVU
+      HUXSUYCUXJUYOXQVUKWQWSWTUYAYTVUCXCAUXEVUNWXJVUMUXCUVJSZVTZVUQUXNXXKUEUFBF
+      VUTVVDXXKUGUHBFVVIWXLVVCUXMUVJSZWXJXXJVVIWXPVVAUXJGSZXWTPZDQWXLXXLVVIWXOX
+      XNDVVIWXNXXMVVPXWTVVIBEGUXJVVAIJVVTVWBVWCYJVVIBEGUXKVVBIJVVTVWIVWHYJWQWSW
+      XRVVIBCDEFGUVJVVAUXJVVBUXKHIJKLMVVTVWAVWCVWBVWEVWDWXQYNWTWXMVVIVUMVVCUXCU
+      XMUVJVVLVVKTWTVWMXCVWNXCYCUVF $.
+  $}
+
+  ${
+    rloc0g.1 $e |- .0. = ( 0g ` R ) $.
+    rloc0g.2 $e |- .1. = ( 1r ` R ) $.
+    rloc0g.3 $e |- L = ( R RLocal S ) $.
+    rloc0g.4 $e |- .~ = ( R ~RL S ) $.
+    rloc0g.5 $e |- ( ph -> R e. CRing ) $.
+    rloc0g.6 $e |- ( ph -> S e. ( SubMnd ` ( mulGrp ` R ) ) ) $.
+    ${
+      rloc0g.o $e |- O = [ <. .0. , .1. >. ] .~ $.
+      $( The identity element of a ring localization.  (Contributed by Thierry
```

Comment:
I think "zero" instead of "identity" is a better term.

Reply:
Fixed in https://github.com/metamath/set.mm/pull/4810/commits/f635662cfffd84458584f2c4a905b88f32b45a51